src/__internal__/validations/validation-icon.style.js

```diff
@@ -20,11 +27,11 @@ const ValidationIconStyle = styled.span`
     `}
 
   ${StyledIcon}:before {
-    color: ${({ validationType, theme }) => theme.colors[validationType]};
+    color: ${({ validationType }) => validationIconTypes[validationType]};
   }
 
   ${StyledIcon}:focus {
-    outline: solid 2px ${({ theme }) => theme.colors.focus};
+    outline: solid 2px var(--colorsSemanticFocus500);
```

Comment:
Did you mean to put this instead of `outline: solid var(--borderWidth300) var(--colorsSemanticFocus500)` - based from the PR description?

Reply:
description doesn't apply to the icon but to the main tab element